lib/exposes.js

```diff
@@ -512,5 +512,13 @@ module.exports = {
             .withFeature(new Numeric('duration', access.SET).withUnit('s').withDescription('Duration in seconds of the alarm')),
         week: () => new Enum('week', access.STATE_SET, ['5+2', '6+1', '7']).withDescription('Week format user for schedule'),
         window_detection: () => new Switch().withState('window_detection', true, 'Enables/disables window detection on the device', access.STATE_SET),
+        acceleration: () => {
```

Comment:
Can we keep these separate? I prefer to keep everything in `presets` the same.

Reply:
Originally i had it in a section called `accelerometer`. 
I figured since it's the same category of "things" it makes sense together.

What do you mean by "keep everything in presets the same" though?

Does that mean after something is added to presets, it shouldn't change?
I suppose that makes sense because other devices that use `moving` via `acceleration` would pick up extra attrs that they might not support.

Reply:
Instead of the `acceleration()` add the 4 presets (moving, x_axis, y_axis, z_axis`) as separate entries instead of grouping them (like all other presets)